terraform/api_gateway.tf.json.template.py

```diff
@@ -206,6 +206,14 @@ def for_domain(cls, domain):
                 }
             }
         },
+        *([{
```

Comment:
Please follow precedent, which is, at least in this module, not to hug open parens/brackets/braces.